sql-join/magic-mallrats.sql

```diff
@@ -0,0 +1,12 @@
+select
+  "firstName",
+  "lastName"
+
+from "rentals"
+
+join "inventory" using ("inventoryId")
+join "films" using ("filmId")
+join "customers" using ("customerId")
+
+where
+  "rentalId" between 2494 and 2495;
```

Comment:
Try targeting the film title `'Magic Mallrats'` with your `where` clause instead of the `"rentalId"`.